### DISCUSSION TITLE
Baby buddy keypad with ESP32 and battery power

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [babybuddy/babybuddy#578] [Show and tell]
Inspired by https://github.com/sfgabe/OITProjects/tree/master/Baby_Buddy_Keypad I made a version of the baby buddy keypad.
Differences between this version and the original apart the physical appearance are:
- Uses ESP32 chip instead of ESP8266 because of option to do deepsleep (for battery version)
- removed light scenes support
- added support for feeding start / end
- added support for tummy time start / end
- added a version that is battery powered using two AA batteries. Since it uses deepsleep it can run for an extended time on just these batteries. Note that if you go this route you only can use single click for the buttons, so no double click/hold/etc.

Read more: 
https://github.com/jeroenterheerdt/Baby-Buddy-Keypad

**Comment by cdubz** [comment]
Thanks for sharing here! I had actually already stumbled across this myself and added it to https://github.com/babybuddy/babybuddy#baby-buddy-on-the-web 😄

**Comment by jeroenterheerdt** [comment]
not seeing my link there, but that's fine. I based my panel on sfgabe's panel (whose link you have in the readme). Was just thinking that adding my link as well made sense as I made several changes, including making it battery powered and switching to ESP32.

**Reply by cdubz** [reply]
Oh derp! It's the other one on there 😄 You gotta lead with your own link :D
